designs/v1-api.md

```diff
@@ -0,0 +1,529 @@
+    - lastTransitionTime: "2024-02-02T19:54:34Z"
+      status: "True"
+      type: Registered
+    - lastTransitionTime: "2024-02-02T19:54:34Z"
+      status: "True"
+      type: Initialized
+    - lastTransitionTime: "2024-02-02T19:54:34Z"
+      status: "True"
+      type: Drifted
+      reason: RequirementsDrifted
+    - lastTransitionTime: "2024-02-02T19:54:34Z"
+      status: "True"
+      type: Expired
+    - lastTransitionTime: "2024-02-02T19:54:34Z"
+      status: "True"
+      type: Empty
+    - lastTransitionTime: "2024-02-02T19:54:34Z"
+      status: "True"
+      type: Ready
+  nodeName: ip-192-168-71-87.us-west-2.compute.internal
+  providerID: aws:///us-west-2b/i-053c6b324e29d2275
+  imageID: ami-0b1e393fbe12f411c
+```
+
+### Printer Columns
+
+**Category:** Stability, Breaking
+
+#### Current
+
+```
+➜  karpenter git:(main) ✗ kubectl get nodeclaims -A -o wide 
+NAME            TYPE         ZONE         NODE                                            READY   AGE    CAPACITY   NODEPOOL   NODECLASS
+default-7lh6k   c6gn.large   us-west-2b   ip-192-168-183-234.us-west-2.compute.internal   True    2d7h   spot       default    default
+default-97v9h   c6gn.large   us-west-2b   ip-192-168-71-87.us-west-2.compute.internal     True    2d7h   spot       default    default
+default-fhzpm   c7gd.large   us-west-2b   ip-192-168-165-122.us-west-2.compute.internal   True    2d7h   spot       default    default
+default-rw4vf   c6gn.large   us-west-2b   ip-192-168-91-38.us-west-2.compute.internal     True    2d7h   spot       default    default
+default-v5qfb   c7gd.large   us-west-2a   ip-192-168-58-94.us-west-2.compute.internal     True    2d7h   spot       default    default
+```
+
+#### Proposed
+
+```
+➜  karpenter git:(main) ✗ kubectl get nodeclaims -A -o wide 
+NAME            TYPE         CAPACITY       ZONE         NODE                                            READY   AGE    ID                                      NODEPOOL  NODECLASS
+default-7lh6k   c6gn.large   spot           us-west-2b   ip-192-168-183-234.us-west-2.compute.internal   True    2d7h   aws:///us-west-2b/i-053c6b324e29d2275   default   default
+default-97v9h   c6gn.large   spot           us-west-2b   ip-192-168-71-87.us-west-2.compute.internal     True    2d7h   aws:///us-west-2a/i-053c6b324e29d2275   default   default
+default-fhzpm   c7gd.large   spot           us-west-2b   ip-192-168-165-122.us-west-2.compute.internal   True    2d7h   aws:///us-west-2c/i-053c6b324e29d2275   default   default
+default-rw4vf   c6gn.large   on-demand      us-west-2b   ip-192-168-91-38.us-west-2.compute.internal     True    2d7h   aws:///us-west-2a/i-053c6b324e29d2275   default   default
+default-v5qfb   c7gd.large   spot           us-west-2a   ip-192-168-58-94.us-west-2.compute.internal     True    2d7h   aws:///us-west-2b/i-053c6b324e29d2275   default   default
+```
+
+**Standard Columns**
+
+1. Name
+2. Instance Type
+3. Capacity - Moved from the wide output to the standard output
```

Comment:
Where in the NodeClaim API does Capacity come from?

Reply:
It comes from the labels -- `karpenter.sh/capacity-type`